shortfin/python/shortfin_apps/llm/components/batcher.py

```diff
@@ -466,6 +464,15 @@ async def get_args(self, bs):
         for page_table in self.page_tables:
             args.append(WrappedAllocation(sfnp.disable_barrier(page_table)))
 
+        updated = True
+        for req in self.exec_requests:
+            if req.allocation is not None:
```

Comment:
We should just sub this out so that we can always call `req.allocation.update_pages` or `req.update_pages`. It is likely we will want to do similar things to handle page copies for deferred page forking.